src/models/mandate.js

```diff
@@ -0,0 +1,33 @@
+import MollieModel from 'mollie-model';
+
+/**
+ * The `Mandate` model
+ */
+export default class Mandate extends MollieModel {
+  static STATUS_VALID = 'valid';
```

Comment:
Static class properties are still in stage-3. Are you sure you'd like to use those already? 
If you would, wouldn't it be better to use the spread operator vs `Object.assign` as well?

Reply:
Stage 3 means the chances of this ever changing are very very small, because both the spec editor and reviewers have signed of on it.

I’m ok with using spread operators since we’re transpiling to ES5 anyway.